src/plugins/plugin.legend.js

```diff
@@ -253,9 +255,9 @@ var Legend = Element.extend({
 					var boxWidth = getBoxWidth(labelOpts, fontSize);
 					var width = boxWidth + (fontSize / 2) + ctx.measureText(legendItem.text).width;
 
-					if (i === 0 || lineWidths[lineWidths.length - 1] + width + labelOpts.padding > minSize.width) {
+					if (i === 0 || lineWidths[lineWidths.length - 1] + width + labelOpts.padding > minSize.width - labelOpts.padding) {
```

Comment:
it's a bit funny to me that we have padding on both sides of the comparison. perhaps it should be

    if (i === 0 || lineWidths[lineWidths.length - 1] + width + 2 * labelOpts.padding > minSize.width) {

Reply:
I agree - the fit logic was not trivial to follow.  My intention was to make as few changes as possible while making this check consistent between vertical/horizontal.  Ideally this fit algo would be refactored in its entirety to be more consistent as a whole between the two.   I didn't really feel that was within the scope of this PR, but I could probably find a happy medium.